tools/c7n_azure/c7n_azure/actions/base.py

```diff
@@ -59,16 +59,20 @@ def process_in_parallel(self, resources, event):
         )
 
     def _log_modified_resource(self, resource, message):
-        template = "Resource '{}' Modified by Custodian Action {}."
+        template = "Resource '{}' Modified by {}."
         name = resource.get('name', 'unknown')
-        id = resource.get('id')
         action = self.__class__.__name__
 
         if message:
             template += ' ' + message
 
         self.log.info(template.format(name, action),
-                      extra={'properties': {'resource_id': id, 'action': action}})
+                      extra=self._get_action_log_metadata(resource))
+
+    def _get_action_log_metadata(self, resource):
+        action = self.__class__.__name__
```

Comment:
I think it is better to use c7n registered action name rather than Python class name